src/main/java/teammates/ui/controller/CreateImageUploadUrlAction.java

```diff
@@ -0,0 +1,40 @@
+package teammates.ui.controller;
+
+import com.google.appengine.api.blobstore.BlobstoreFailureException;
+
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.util.Const;
+import teammates.common.util.GoogleCloudStorageHelper;
+
+public class CreateImageUploadUrlAction extends Action {
+
+    @Override
+    protected ActionResult execute() throws EntityDoesNotExistException {
+        return createAjaxResult(getCreateImageUploadUrlPageData());
+    }
+
+    protected final CreateImageUploadUrlAjaxPageData getCreateImageUploadUrlPageData() {
+        verifyPrivileges();
```

Comment:
Actually, why does `verifyPrivileges()` part of `get*PageData()` method? If this method is narrated, it would be something like:
```
I want to get the `*PageData`, so I first verify if I have the privileges, then ...
```
While not entirely wrong, consider moving it to `execute()` instead. The narration becomes:
```
I want to execute this action, so I first verify if I have the privileges, then ...
```
Which makes much more sense. It also removes the need for you to have an empty `verifyPrivileges()` method. This also applies for one in `ImageUploadAction`.